### ISSUE TITLE
[Android App 1.6.10] Stremio interface and on-screen controls on Android are extremely sluggish, almost unresponsive, during playback with Real Debrid

### ISSUE THREAD

**Issue by asdrubale88** [Stremio/stremio-bugs#671]
**Describe the bug**
Using the Stremio app on Android (latest version, Stremio 1.6.10) during playback with Real Debrid the interface is extremely sluggish, almost unresponsive, when playing a video with AND without the background service active as well. I noticed that the slower the CPU of the device, the more this unresponsiveness is accentuated.

_It seems that this issue is CPU related, as faster tablets will show less lag/unresponsiveness, but still some with respect to any local player._

**To Reproduce**
1. Go to a Movie/TV Series
2. Click on any Episode and start it
3. Touch the display to bring up the seek bar and controls
4. Touch the seek bar or any control on screen, the command is not registered correctly
5. You need to keep tapping any control and hope that the command is registered and the interface reacts accordingly

**Expected behavior**
I'd expect the interface to be responsive and smooth.

**Android Tablet Samsung S5E:**
 - OS: Android 11 - One UI 3.1
 - Stremio Android app
 - Version 1.6.10
 - Note: Android app is VERY sluggish, almost unresponsive (note that the CPU of this tablet is SLOW).

**Android Tablet Xiaomi Mi Pad 6:**
 - OS: Android 13 - MIUI Pad Global 14.0.8
 - Stremio Android app
 - Version 1.6.10
 - Note: Android app is somewhat usable, but sluggish nonetheless.

**Additional context**
It seems that using the Web version of Stremio (web.stremio.com) is a workaround in the sense that it works smoothly even on the old Samsung tablet.

EDIT: A minor update to Stremio Android (v1.6.10) was released yesterday, but the issue is still there.

**Comment by asdrubale88** [comment]
I noticed also this: if the video is not playing, everything is pretty responsive.
It doesn't seem that the video file type or its size matters, all on screen commands become almost unresponsive.

**Comment by krymzone** [comment]
Can confirm, I hame the same problem on Xiaomi Redmi Pad SE, the buttons are very sluggish and unresponsive. The problem seems to mostly occur on tablets, I have no problem with the app on my phone.

**Comment by T1337Dg** [comment]
This issue still happens in version 1.6.12 on Samsung A31

**Comment by krymzone** [comment]
> Can confirm, I have the same problem on Xiaomi Redmi Pad SE, the buttons are very sluggish and unresponsive. The problem seems to mostly occur on tablets, I have no problem with the app on my phone.

Edit: It still occurs on my Redmi Pad, but now it also happens on my phone too (Samsung A51), the issue seems to happen only to some videos though, not all of them, the only pattern i managed to notice is that it mostly happens with MKV, MP4 doesn't seem to have this problem (or it's simply really rare)

**Comment by asdrubale88** [comment]
@krymzone I agree with you, it's probably not due to video/file size but the format. I suspect some video formats are CPU decoded, hence the CPU decoding is taking precedence over the interface of the Stremio GUI and that is causing the sluggishness. Maybe some GPU codecs are missing, maybe some older chipsets do not support AV1/VP9 or other formats hardware decoding.

**Comment by TRtomasz** [comment]
Closing issue as new app version is available, right now through betta channel but this will be main focus now.
